tests/js/spec/views/organizationGroupDetails/__snapshots__/groupSimilar.spec.jsx.snap

```diff
@@ -1685,7 +1685,7 @@ exports[`Issues Similar View renders with mocked data 1`] = `
                                 vertical={true}
                               >
                                 <ScoreBar
-                                  className="css-tics7q-StyledScoreBar eecxaw40"
+                                  className="css-tics7q-StyledScoreBar e1h9dyli0"
```

Comment:
Included the snapshot changes, are these changing because of functional changes or is because I modified the file?

Reply:
You changed a property that changed a CSS definition. Jest snapshots + emotion = sad times.

Reply:
Are we considering using something like https://github.com/emotion-js/emotion/tree/master/packages/jest-emotion so it's not just random class names?

Reply:
Not using snapshots is the approach we've been using so far. They have historically not provided us a ton of value.